distribution/changelog.txt

```diff
@@ -83,6 +83,7 @@
 - Fix: [#9411] Ad campaigns end too soon.
 - Fix: [#9476] Running `simulate` command on park yields `Completed: (null)`.
 - Fix: [#9520] Time Twister object artdec29 conversion problem.
+- Fix: [#9675] Guest entry point limit can be bypassed in scenario editor
```

Comment:
Please add a full stop at the end.

Reply:
I also added the entry to the wrong section!